mithril-aggregator/src/store/epoch_settings_storer.rs

```diff
@@ -37,7 +36,6 @@ pub trait EpochSettingsStorer: Sync + Send {
         for epoch_offset in 0..=2 {
             let epoch = current_epoch + epoch_offset;
             if self.get_epoch_settings(epoch).await?.is_none() {
-                debug!("Handle discrepancies at startup of epoch settings store, will record epoch settings from the configuration for epoch {epoch}: {epoch_settings_configuration:?}");
```

Comment:
Do we want to remove this log?

Reply:
It has not been removed but moved to where it was called in the builder. I can't keep it here without a global log system such as `slog_scope`.